src/main/java/org/sonar/plugins/github/GitHubPlugin.java

```diff
@@ -58,7 +58,15 @@
     type = PropertyType.INTEGER,
     global = false,
     project = false,
-    module = false)
+    module = false),
+  @Property(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder annotation arguments to match the order of declaration. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS3340)